journals/apps/journals/templates/admin/import_users.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'admin/base.html' %}
+{% load i18n admin_modify formadmin_filters admin_urls%}
+
+{% block title %} {% trans "Import Users" %} {% endblock %}
+
+{% block extrastyle %}
+    <link rel="stylesheet" type="text/css" href="/static/admin/css/forms.css">
+{% endblock %}
+{% block branding %}
+    <h1 id="site-name"><a href="{% url 'admin:index' %}"> {% trans "Django administration" %}</a></h1>
+{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+  <a href="{% url 'admin:index' %}">{% trans "Home" %}</a>
+  &rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+  &rsaquo; {% if has_change_permission %}
+    <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+  {% else %}
+    {{ opts.verbose_name_plural|capfirst }}
+  {% endif %}
+  &rsaquo;
+   {% trans "Import Users" %}
+
+</div>
+{% endblock %}
+
+
+{% block content %}
```

Comment:
I think we should just override block content instead of copying over everything.